tests/unit/list/slist.c

```diff
@@ -202,6 +202,14 @@ ZTEST(dlist_api, test_slist)
 				       &test_node_1.node, true)),
 		     "test_list head/tail are wrong");
 
+	/* Find the node 1, previous node should be null */
+	sys_snode_t *test_node_1_prev = NULL;
```

Comment:
It might be a good idea to set this to a value other than null so that we can verify that the function modified it